datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -411,40 +411,35 @@ where
         Box::new(
             // Get a stream of all relevant logs in range
             self.log_stream(from, to, log_filter)
-
                 // Get first chunk of logs
                 .take(1)
-
                 // Collect 0 or 1 vecs of logs
                 .collect()
-
```

Comment:
hmmm

Reply:
Seems rustfmt is now against blank lines between method calls.

Reply:
not a fan, but I will cope :smile: